src/prefs.js

```diff
@@ -0,0 +1,64 @@
+const Gtk = imports.gi.Gtk;
+
+let Extension = imports.misc.extensionUtils.getCurrentExtension();
+let Settings = Extension.imports.settings;
+
+let catSleepingSlider = null;
+
+function init() { }
+
+function buildPrefsWidget() {
```

Comment:
If you make it more OOP-styled -- will be great. If not, it's also fine, because I'm working in [glade files](https://glade.gnome.org/) support, this file will be updated too

Reply:
> I think i wouldn't have time to update the pref.js, i'm trying to see if i can update this weekend if not can you do it?

@bouteillerAlan, Do you mind if I do it for you?

Reply:
I've made changes